frequency: 1
# 0 = default(list all), 1 = whitelist, 2 = blacklist
filter_type: 1

monitor_mode: 5
#AggregationStrategies mode: 1 LAST, 2 FIRST, 3 MIN, 4 MAX, 5 AVG#
